controllers/kyma_controller_helper_test.go

```diff
@@ -237,6 +239,21 @@ func ModuleTemplatesExist(clnt client.Client, kyma *v1beta1.Kyma, remote bool) f
 	}
 }
 
+func WatcherLabelsAnnotationsExist(clnt client.Client, kyma *v1beta1.Kyma) error {
+	remoteKyma, err := GetKyma(ctx, clnt, kyma.GetName(), kyma.Spec.Sync.Namespace)
```

Comment:
```suggestion
	remoteKyma, err := GetKyma(ctx, clnt, kyma.GetName(), kyma.GetNamespace())
```
so that I don't have to refactor again because of this issue https://github.com/kyma-project/lifecycle-manager/issues/544